components/brave_wallet_ui/page/screens/send/components/account-selector/account-selector.tsx

```diff
@@ -60,7 +60,14 @@ export const AccountSelector = (props: Props) => {
 
   // Memos
   const accountsByNetwork = React.useMemo(() => {
-    return accounts.filter((account) => account.coin === selectedNetwork?.coin && account.keyringId === selectedAccount?.keyringId)
+    if (!selectedNetwork || !selectedAccount) {
+      return []
+    }
+    return accounts.filter(
+      (account) =>
+        account.accountId.coin === selectedNetwork.coin &&
+        account.accountId.keyringId === selectedAccount.accountId.keyringId
+    )
```

Comment:
If a future follow-up, would be nice to have a util that compares the equivalence of two `AccountId` objects.